recipes/slack.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/* (c) Tomas Majer <tomasmajer@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Notify Slack of successful deployment
+ */
+task('deploy:slack', function () {
+    global $php_errormsg;
```

Comment:
I Think @elfet is referring to php_errormsg, which is used to create the runtime exception.
Also, this recipe along with newrelic can throw an exception, which if thrown will break the deploy, even though it was successful. 
There should be an option to display non fatal errors. 
I proposed https://github.com/deployphp/deployer/pull/108 but it wasn't a good solution.